release_notes.md

```diff
@@ -3,17 +3,10 @@
 <!-- Please add your release notes in the following format:
 - My change description (#PR)
 -->
-- Update Python Worker Version to [4.16.0](https://github.com/Azure/azure-functions-python-worker/releases/tag/4.16.0)
-- Update Java Worker Version to [2.12.1](https://github.com/Azure/azure-functions-java-worker/releases/tag/2.12.1)
```

Comment:
As far as I can tell, the only line that should change is the Python worker line - downgrading it to 4.14.0. Can we make sure that is the only change in this PR?